ui/react-components/components/PatientSearch/PatientSearch.jsx

```diff
@@ -33,6 +33,7 @@ const PatientSearch = (props) => {
             placeholder={placeholder}
             selectedValue={value}
             isDisabled={isDisabled}
+            autoFocus={true}
```

Comment:
autoFocus should come from parent, we can use PatientSearch somewhere else and not have the first focus on it.

Reply:
Yes, handled the same